app/entrypoint.sh

```diff
@@ -1,10 +1,18 @@
 #!/bin/sh
 
```

Comment:
The code to handle signals properly didn't end up being too arcane and is more legible with some sensible comments.  I'll take it out again if/when we switch to using tini but for now it doesn't hurt to be more correct about how we handle zombie process reaping.